letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
what's the problem with leaving everything as it was and injecting a proxy header where necessary? I really don't want to introduce hacks here that lead the multi database stuff ad absurdum.